workspace/popjava/src/popjava/dataswaper/POPByte.java

```diff
@@ -0,0 +1,76 @@
+package popjava.dataswaper;
+
+import popjava.buffer.POPBuffer;
+/**
+ * Primitive settable byte for POPJava, needed for {@link popjava.annotation.POPParameter} direction use in methods 
```

Comment:
I removed the `POPFloat` but it doesn't seems to work as expected with primitive floats. Even with `Direction.INOUT`
```
float fitness = 0;
resIDs[jobIdx] = jm.reserve(od, fitness, appId, reqId);
```
`fitness` is always 0, even if `reserve` modify it.